packages/design-system/src/components/Alert/Alert.stories.jsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+
+import Alert from './Alert';
+
+export default {
+  title: 'Components/Alert',
+  component: Alert,
+  argTypes: {},
+  args: {
+    hideIcon: false,
+  },
+};
+
+const Template = (args) => (
```

Comment:
There's examples of Alerts without headings in the docs that should probably be represented. I don't know if you've heard otherwise though.

Reply:
No just shooting from the hip with these, that sounds good i'll add one without heading for each type to round them out 👍

Reply:
Added an alert without a heading to the default template